Formula/proxsuite.rb

```diff
@@ -0,0 +1,50 @@
+class Proxsuite < Formula
+  desc "Advanced Proximal Optimization Toolbox"
+  homepage "https://github.com/Simple-Robotics/proxsuite"
+  url "https://github.com/Simple-Robotics/proxsuite/releases/download/v0.2.2/proxsuite-0.2.2.tar.gz"
+  sha256 "da72d900c86fe77a1572f4f2866600ef56a63bd72f050206cc0a2c1643c8f185"
+  license "BSD-2-Clause"
+  head "https://github.com/Simple-Robotics/proxsuite.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "pkg-config" => :build
+  depends_on "eigen"
+  depends_on "numpy"
+  depends_on "python@3.10"
+  depends_on "scipy"
+  depends_on "simde"
+
+  def python3
+    Formula["python@3.10"].opt_libexec/"bin/python"
+  end
+
+  def install
+    if build.head?
+      system "git", "submodule", "update", "--init"
+      system "git", "pull", "--unshallow", "--tags"
+    end
+
+    ENV.prepend_path "PYTHONPATH", Formula["eigenpy"].opt_prefix/Language::Python.site_packages(python3)
+    ENV.prepend_path "Eigen3_DIR", Formula["eigen"].opt_share/"eigen3/cmake"
+
+    system "cmake", "-S", ".", "-B", "build",
+                    "-DPYTHON_EXECUTABLE=#{python3}",
+                    "-DBUILD_UNIT_TESTS=OFF",
+                    "-DBUILD_PYTHON_INTERFACE=ON",
+                    "-DINSTALL_DOCUMENTATION=ON",
+                    "-DCMAKE_OSX_DEPLOYMENT_TARGET=10.15",
+                    "-DSimde_INCLUDE_DIR=#{Formula["simde"].opt_prefix/"include"}",
```

Comment:
Why doesn't CMake detect this correctly?

Reply:
I don't know. But this seems to not work here. Is it a problem?

Reply:
Can you file an issue upstream to make them aware of this?

Reply:
Yes, will do.

Reply:
Still waiting for that issue

Reply:
I've just opened an issue on this here: https://github.com/Simple-Robotics/proxsuite/issues/65